src/fontra/backends/designspace.py

```diff
@@ -682,14 +680,17 @@ def buildUFOLayerGlyph(
         if component.location:
             # It's a variable component
             varCoDict = {"base": component.name, "location": component.location}
-            if component.transformation != Transformation():
+            if isinstance(component.transformation, DecomposedTransform):
                 varCoDict["transformation"] = asdict(component.transformation)
             variableComponents.append(varCoDict)
         else:
             # It's a regular component
+            decomposedTransform = DecomposedTransform(
```

Comment:
I don't think this should be necessary: `component.transform` should already be a `DecomposedTransform` instance.

Reply:
Ok, fixed.